spark/dpl-interpreter/src/test/java/com/teragrep/pth_07/ui/elements/table_dynamic/DTTableDatasetNgTest.java

```diff
@@ -228,7 +228,15 @@ public void testClearParagraphResultsOnNewDataset(){
             dtTableDatasetNg.setParagraphDataset(testDs);
         });
         Assertions.assertEquals(1,listener.numberOfUpdateCalls());
-        Assertions.assertEquals(1,listener.numberOfResetCalls());
+        Assertions.assertEquals(0,listener.numberOfResetCalls());
+
+        // Simulate DPL receiving another batch of data.
+        Assertions.assertDoesNotThrow(()->{
+            dtTableDatasetNg.setParagraphDataset(testDs);
+        });
+        Assertions.assertEquals(2,listener.numberOfUpdateCalls());
+        Assertions.assertEquals(0,listener.numberOfResetCalls());
+
     }
```

Comment:
Slight suggestion for formatting

```suggestion
        // Simulate DPL receiving another batch of data.
        Assertions.assertDoesNotThrow(()->{
            dtTableDatasetNg.setParagraphDataset(testDs);
        });
        
        Assertions.assertEquals(2,listener.numberOfUpdateCalls());
        Assertions.assertEquals(0,listener.numberOfResetCalls());
}
```